dialog/base.go

```diff
@@ -245,27 +246,28 @@ func newButtonList(buttons ...*widget.Button) fyne.CanvasObject {
 	return list
 }
 
-// dialogTitle is really just a normal title but we use the Refresh() hook to update the background rectangle.
-type dialogTitle struct {
-	widget.Label
+// TODO I don't think it is needed
```

Comment:
I agree that this is not needed.
But please don't commit commented out code.

Reply:
Sorry, I will remove it.

Reply:
done